azure-iot-hub/samples/digital_twin_service_client_sample.py

```diff
@@ -0,0 +1,56 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import os
+from azure.iot.hub import DigitalTwinServiceClient
+
+connection_str = os.getenv("IOTHUB_CONNECTION_STRING")
+device_id = "<DEVICE_ID_GOES_HERE>"  # Existing device's id
+interface_instance_name = (
+    "<INTERFACE_INSTANCE_NAME_GOES_HERE>"
+)  # for the environmental sensor, try "environmentalSensor"
+model_id = "<MODEL_ID_GOES_HERE>"  # Existing model's id
+
+try:
+    # DigitalTwinServiceClient
+    digital_twin_service_client = DigitalTwinServiceClient(connection_str)
+    # GetDigitalTwin
+    digital_twin = digital_twin_service_client.get_digital_twin(device_id)
+    print("DigitalTwin: ")
+    print(digital_twin)
+
+    # GetDigitalTwinInterfaceInstance
```

Comment:
>GetDigitalTwinInterfaceInstance [](start = 6, length = 31)

Not sure I quite understand what these comments are.

Reply:
Sure, changed to descriptive ones

---
In reply to: [322933786](https://github.com/Azure/azure-iot-sdk-python/pull/300#discussion_r322933786) [](ancestors = 322933786)